storage/tianmu/handler/ha_tianmu.h

```diff
@@ -162,7 +162,7 @@ class ha_tianmu final : public handler {
 
```

Comment:
dupkey_pos of uint is a posite value  assign to a negative value -1, better to assign it to UINT32_MAX (defined in stdint.h)

Reply:
Fixed.

Reply:
Fixed.